README.md

```diff
@@ -8,7 +8,7 @@ Azure Terrafy imports the resources that are supported by the [Terraform AzureRM
 
 ## Non Goal
 
-The Terraform configurations generated by `aztfy` is not meant to be comprehensive. This means it doesn't guarantee the infrastruction can be reproduced via the generated configurations. For details, please refer to the [limitation](#limitation).
+The Terraform configurations generated by `aztfy` are not meant to be comprehensive and do not ensure that the infrastructure can be fully reconstructed from the generated configurations. For details, please refer to the [limitation](#limitation).
```

Comment:
Can we put the word "reproduce" here to mention that the idea of "reproducible provision" is not guaranteed by this tool?

```suggestion
The Terraform configurations generated by `aztfy` are not meant to be comprehensive and do not ensure that the infrastructure can be fully reproduced from the generated configurations. For details, please refer to the [limitation](#limitation).
```